packages/memory/src/index.ts

```diff
@@ -150,18 +154,22 @@ export class MemoryDriver extends Driver {
 
   async upsert(sel: Selection.Mutable, data: any, keys: string[]) {
     const { table, model, ref } = sel
+    const result: Driver.WriteResult = { inserted: 0, modified: 0 }
```

Comment:
```suggestion
    const result = { inserted: 0, modified: 0 }
```

Then you could omit the `!` in the following code.